src/effects/VST3/VST3Effect.cpp

```diff
@@ -1129,6 +1129,8 @@ bool VST3Effect::LoadPreset(const wxString& path)
    if(mPlainUI != nullptr)
       mPlainUI->ReloadParameters();
 
+   FetchSettings(GetSettings(settings));
```

Comment:
The commit that inserts the `FetchSettings` calls is good.  It should be squashed with the earlier commit that rewrites `LoadSettings` and `SaveSettings`.  That commit isn't complete without the calls to `FetchSettings` because `SaveSettings` is assuming that the fetch was already done, and is not correct without it.

Reply:
ok, done